### PR TITLE
Remove irrelevant "gfx.offscreencanvas.enabled" flag in Firefox Android

### PR DESCRIPTION
**Pull request by queengooborg**
This PR removes irrelevant flag data for the `gfx.offscreencanvas.enabled` flag of Firefox Android as per the corresponding [data guidelines](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#removal-of-irrelevant-flag-data).

### CODE REVIEW

File: api/OffscreenCanvas.json [hunk 1]
```diff
@@ -213,7 +188,7 @@
                 "version_added": "96"
               },
               "firefox_android": {
-                "version_added": "96"
+                "version_added": false
```
Comment (queengooborg) [inline api/OffscreenCanvas.json:191]: This change may seem irrelevant, but this is required for consistency.

**Review by foolip — APPROVED** [review]
